syscall/devices/hostfs/hostfs.c

```diff
@@ -433,6 +433,9 @@ static int _hostfs_dup(oe_fd_t* desc, oe_fd_t** new_file_out)
     file_t* file = _cast_file(desc);
     file_t* new_file = NULL;
 
+    if (!new_file_out)
+        OE_RAISE_ERRNO(OE_EINVAL);
+
     if (new_file_out)
```

Comment:
I believe line 439 can be removed since with the new code above, this line cannot be reached unless new_file_out is non-null